src/jit/codegen.h

```diff
@@ -464,7 +464,8 @@ protected :
                                     INDEBUG_LDISASM_COMMA(CORINFO_SIG_INFO* sigInfo)
                                     void*                 addr
                                     X86_ARG(ssize_t       argSize),
-                                    emitAttr              retSize,
+                                    emitAttr              retSize
+                                    FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(emitAttr retSize1),
```

Comment:
this is a nit, but wouldn't it be easier if the 'retSize' line still had a comma, and this was:

FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY(emitAttr retSize1,)